packages/webapp/src/components/Signup/EnterPasswordPage/index.js

```diff
@@ -44,7 +47,9 @@ export default function PureEnterPasswordPage({ title = 'Welcome back', onLogin
         </>
       }
     >
-      <Title style={{ marginBottom: '32px' }}>{title}</Title>
+      <Title style={{ marginBottom: '32px' }}>
+        {title + ',' + ' ' + userName.userName.userName}
```

Comment:
Having `userName.userName.userName` does not look good. This leads me to find that the `manualSignUpReducer` is being set incorrectly (with nested objects) when an email is being input. Fix this and we won't need to use `userName.userName.userName`.

Reply:
Done.